src/kubernetes/installer.cr

```diff
@@ -183,6 +183,7 @@ class Kubernetes::Installer
       cluster_dns: settings.networking.cluster_dns,
       datastore_endpoint: datastore_endpoint,
       etcd_arguments: etcd_arguments,
+      s3_arguments: generate_s3_arguments(),
```

Comment:
Similarly to Ruby, there is no need for `()` when invoking a method without any arguments :)

Reply:
Also I'd rename this to `etcd_backup_settings` or `etcd_snapshot_settings` to make it clear that these settings are for etcd backups/snapshots.